client/src/components/ButtonSeason.js

```diff
@@ -0,0 +1,32 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+ButtonSeason.propTypes = {
+  onClick: PropTypes.func.isRequired,
+  isActive: PropTypes.bool.isRequired,
+}
+
+export default function ButtonSeason({ name, onClick, isActive }) {
+  return (
+    <StyledButton onClick={onClick} isActive={isActive}>
```

Comment:
you could say `onClick={handleButtonSeasonClick}` to give it a bit more specificity for future you :)